x/group/keeper/keeper_test.go

```diff
@@ -806,7 +848,6 @@ func (s *TestSuite) TestCreateGroupPolicy() {
 			s.Assert().Equal(spec.req.Admin, groupPolicy.Admin)
 			s.Assert().Equal(spec.req.Metadata, groupPolicy.Metadata)
 			s.Assert().Equal(uint64(1), groupPolicy.Version)
-			s.Assert().Equal(spec.policy.(*group.ThresholdDecisionPolicy), groupPolicy.GetDecisionPolicy())
```

Comment:
we should still verify this but adapt it to either make the type assertion on `ThresholdDecisionPolicy` or `PercentageDecisionPolicy`